testdata/p4_16_samples_outputs/bitvec_hdr.p4

```diff
@@ -0,0 +1,43 @@
+struct macDA_t {
+    bit<28> lower28Bits;
+    bit<20> upper20Bits;
+}
+
+struct sip_t {
+    bit<28> lower28Bits;
+    bit<4>  upper4Bits;
+    macDA_t macDA;
+}
+
+header our_bitvec_header {
+    bit<2> g2;
+    bit<3> g3;
+    sip_t  sip;
+}
+
+typedef bit<3> Field;
+header your_header {
+    Field field;
+}
+
+header_union your_union {
+    your_header h1;
+}
+
+struct str {
+    your_header hdr;
+    your_union  unn;
+    bit<32>     dt;
+}
+
+control p() {
```

Comment:
because this program has no main it does not go through the whole front-end. Please define a package and instantiate it.

Reply:
Will do and included changes in next commit.